schema_utils.py

```diff
@@ -94,6 +94,10 @@ def normalize_against_schema(
     normalized_obj = None # type: Any
 
     if schema[SCHEMA_KEY_TYPE] == SCHEMA_TYPE_BOOL:
+        if obj == 'true':
+            obj = True
```

Comment:
@DubeySandeep I want to confirm that whether this approach is fine or we should leave **SCHEMA_TYPE_BOOL** as it is and handle the cases for 'true'/'false' by **SCHEMA_TYPE_BASESTRING**.